test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -338,8 +341,10 @@ public void ParseValue ()
             Assert.Equal("a#b#c", Parsers.Value.AtEnd().Parse("a#b#c").Value);
             Assert.Equal("041", Parsers.Value.AtEnd().Parse("041").Value);
             Assert.Equal("日本", Parsers.Value.AtEnd().Parse("日本").Value);
-            // TODO: is it possible to get the system to recognize when a complete unicode char is present and start the next one then, without a space?
-//            Assert.Equal("日本", Parsers.Value.AtEnd().Parse(@"\xe6\x97\xa5\xe6\x9c\xac"));
+
+            Assert.Equal(@"\xe6\x97\xa5\xe6\x9c\xac", Parsers.Value.AtEnd().Parse(@"\xe6\x97\xa5\xe6\x9c\xac").Value);
+            Assert.Equal(@"\xe6\x97\xa5\xe6\x9c\xac", Parsers.Value.AtEnd().Parse(@"'\xe6\x97\xa5\xe6\x9c\xac'").Value);
+            Assert.Equal("日本", Parsers.Value.AtEnd().Parse(@"""\xe6\x97\xa5\xe6\x9c\xac""").Value);
```

Comment:
Minor note: as you can see in later examples in this test, I prefer `\"` instead of `""`

Reply:
done